checks/raw/license.go

```diff
@@ -69,16 +69,49 @@ func License(c *checker.CheckRequest) (checker.LicenseData, error) {
 	var results checker.LicenseData
 	var path string
 
```

Comment:
you also need to update https://github.com/ossf/scorecard/blob/6f84cb92bb88aef0db98813f127f3981f74775c4/checks/license.go#L29-L31 since the check no longer support file based (it needs the API).

Reply:
I'll look into that, I did not know that, I may circle back and ask questions if I have difficulty.

Reply:
done.